lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -468,6 +502,7 @@ def __init__(
         container_name: str,
         cap_net_admin: bool = False,
         privileged: bool = False,
+        refresh_event: Optional[Union[BoundEvent, List[BoundEvent]]] = None,
```

Comment:
Why make it optional? My recommendation is to have it simply be of type BoundEvent. No need to support list of events at the moment, afaik there is no use case for this right now. This should make the rest of the init quite simple:

```python
self.framework.observe(refresh_event, self._configure_multus)
```

Reply:
I tried to support the single Bound event or list of Bound events by considering the future requirements. At the moment, single event is enough. We can support list of events when we need.